pkg/auth.go

```diff
@@ -37,46 +37,40 @@ const PublicURL = "publicUrl"
 // ConfMode is the config name for the engine mode
 const ConfMode = "mode"
 
-// ConfIrmaConfigPath is the config key to provide the irma configuration path
-const ConfIrmaConfigPath = "irmaConfigPath"
-
 const ConfEnableCORS = "enableCORS"
 
 // ConfActingPartyCN is the config key to provide the Acting party common name
 const ConfActingPartyCN = "actingPartyCn"
 
-// ConfIrmaSchemeManager allows selecting an IRMA scheme manager. During development this can ben irma-demo. Production should be pdfb
-const ConfIrmaSchemeManager = "irmaSchemeManager"
-
 // AuthClient is the interface which should be implemented for clients or mocks
 type AuthClient interface {
-	CreateContractSession(sessionRequest types2.CreateSessionRequest) (*types2.CreateSessionResult, error)
-	ContractSessionStatus(sessionID string) (*types2.SessionStatusResult, error)
+	CreateContractSession(sessionRequest types.CreateSessionRequest) (*types.CreateSessionResult, error)
+	ContractSessionStatus(sessionID string) (*types.SessionStatusResult, error)
 	ContractByType(contractType contract.ContractType, language contract.Language, version contract.Version) (*contract.ContractTemplate, error)
-	ValidateContract(request types2.ValidationRequest) (*types2.ContractValidationResult, error)
-	CreateAccessToken(request types2.CreateAccessTokenRequest) (*types2.AccessTokenResponse, error)
-	CreateJwtBearerToken(request types2.CreateJwtBearerTokenRequest) (*types2.JwtBearerTokenResponse, error)
-	IntrospectAccessToken(token string) (*types2.NutsAccessToken, error)
+	ValidateContract(request types.ValidationRequest) (*types.ContractValidationResult, error)
+	CreateAccessToken(request types.CreateAccessTokenRequest) (*types.AccessTokenResponse, error)
```

Comment:
`ContractValidationResult` and `AccessTokenResponse` (`Result` vs `Response`). And later on I see just `NutsAccessToken`.

I think `Result` is fine (since it's the 'result' of a process/calculation), but `Response` indicates a client-server or other type of call over a network to me. Since this is just an interface which can be (and is) called directly I don't think `Response` is appropriate. I'd expect to see `Response` structs in the HTTP layer.

Reply:
The usages of `Response` is inspired on the messages pattern, but its usage is currently very inconsistent indeed. Lets change that.